src/tic_tac_toe/board.clj

```diff
@@ -1,4 +1,13 @@
 (ns tic-tac-toe.board)
 
-(defn empty-board []
-  '(- - - - - - - -))
+(defn- valid-location? [location board] 
+  (if (<= location (- (count board) 1))
+     true
+     false))
+
+(defn make-move [board location mark]
+  (if (valid-location? location board)
```

Comment:
if you give the board an invalid move, will it just quietly ignore it? Maybe the game should check if a move is valid before it adds to the board?